terminal_client.go

```diff
@@ -119,7 +119,7 @@ func (c *TerminalClient) listCmd() error {
 
 	tbl := TableDef{
 		Columns: []ColumnDef{
-			ColumnDef{"id", 25},
+			ColumnDef{"id", 42},
```

Comment:
I think it would, because messages might contain a lot of stuff and a trailing active indicator could get lost in the noise / wrapped onto a new line etc